.buildkite/scripts/trigger_integrations_in_parallel.sh

```diff
@@ -88,6 +88,11 @@ for package in ${PACKAGE_LIST}; do
         FORCE_CHECK_ALL: "${FORCE_CHECK_ALL}"
         SERVERLESS: "false"
         UPLOAD_SAFE_LOGS: ${UPLOAD_SAFE_LOGS}
+      plugins:
+        # See https://github.com/elastic/oblt-infra/blob/main/conf/resources/repos/integrations/01-aws-buildkite-oidc.tf
+        # This plugin creates the environment variables required by the service deployer (AWS_SECRET_ACCESS_KEY and AWS_SECRET_KEY_ID)
+        - elastic/oblt-aws-auth#v0.1.0:
+            duration: 3600 # seconds
```

Comment:
`2 hours`, no clue if steps take longer

```suggestion
            duration: 7200 # seconds
```